src/shared.js

```diff
@@ -253,9 +274,12 @@ export const performSearch = async (query) => {
     showAnimation();
     const response = await findParticipant(query);
     hideAnimation();
-    const getVerifiedParticipants = response.data.filter(dt => dt['821247024'] === 197316935);
-    if(response.code === 200 && getVerifiedParticipants.length > 0) searchResults(getVerifiedParticipants);
-    else if(response.code === 200 && getVerifiedParticipants.length === 0) showNotifications({title: 'Not found', body: 'The participant with entered search criteria not found!'}, true)
+    const verifiedParticipants = response.data.filter(dt => dt['821247024'] === 197316935);
+
+    if (response.code === 200 && verifiedParticipants.length > 0) {searchResults(verifiedParticipants);
```

Comment:
this is so small... but can you structure the code within the brackets for the IF statement similar to what you did with the ELSE IF statement? (code is in line between the brackets)

Reply:
Make it beautiful. Sure.

Reply:
thank you 🥇